tools/targets/lint.py

```diff
@@ -29,6 +29,7 @@
 from copy import copy
 from yaml import dump_all
 import argparse
+from past.builtins import basestring
```

Comment:
is basestring being used? if so, please ignore this comment

Reply:
I can't ignore your comments 😄 Yeah its used later in the file.